packages/repco-webcomponent/src/post-card/post-card-body.ts

```diff
@@ -0,0 +1,27 @@
+import {css, html, LitElement} from 'lit';
+import {customElement, property} from 'lit/decorators.js';
+import {unsafeHTML} from 'lit/directives/unsafe-html.js';
+
+@customElement('post-card-body')
+export class PostCardBodyElement extends LitElement {
+  static override styles = css`
+    :host {
+      flex-grow: 2;
+      overflow: hidden;
+      text-overflow: ellipsis;
+      color: var(--medium-body-color);
+    }
+  `;
+
+  @property()
+  body = '';
+  override render() {
+    return html`${unsafeHTML(this.body)}`;
```

Comment:
this is .. unsafe ;) we likely need to check the HTML there. or only allow strings not html. we cannot inject any HTML coming from the repco node into the DOM, this opens it up to XSS etc vulnerabilities.

this is a bit of a bigger issue that we'll have to address in repco. for now, we should either sanitze the HTML here or convert it to string only (remove all html tags).

Reply:
See https://github.com/openaudiosearch/repco/issues/61